packages/app/src/cli/models/extensions/specification.ts

```diff
@@ -22,8 +22,8 @@ export interface TransformationConfig {
 }
 
 export interface CustomTransformationConfig {
-  forward?: (obj: object) => object
-  reverse?: (obj: object) => object
+  forward?: (obj: object, options?: {flags?: Flag[]}) => object
```

Comment:
Currently `transform` doesnt support the `options` params so maybe we don't need to add this param to the `forward` method

Reply:
Do you mean that we are not using the `options` param in the current forward implementations? I thought about that, but I think it's better to keep the same shape for both methods. WDYT?